benchexec/container.py

```diff
@@ -722,6 +740,37 @@ def escape(s):
     )
 
 
+def make_fuse_overlay_mount(exe, mount, lower, upper, work):
+    logging.debug(
+        "Creating overlay mount: target=%s, lower=%s, upper=%s, work=%s",
```

Comment:
How about adding an indicator that it is fuse-based?

Reply:
Maybe like `Creating overlay mount with fuse-overlayfs: target=%s, lower=%s, upper=%s, work=%s`?